app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ReservationsController < ApplicationController
+  def create
+    @reservation = Reservation.new(item_id: params[:item_id])
+    return unless @reservation.save
```

Comment:
We don't need new method because be don't have any form, so for now there's button in item page that only passes item_id to Reservation model.